index.bs

```diff
@@ -63,7 +63,7 @@ that additional functionality is required.
 
 The additional functionality documented herein aims to address:
 
-1. Resource servers having no existing trust relationship with identity providers.
+1. Resource servers and their Authorization servers having no existing trust relationship with identity providers.
```

Comment:
A general comment is that this is not strictly true. Since the AS fetches the WebID document of the user it relies on being able to trust the TLS certificate of the WebID URL. This implies that the AS shares a common trust-anchor with the OP. This assumption should at the very least be called out.

Reply:
It is true that the AS may share the same the same trust-anchor as the OP w.r.t the TLS certificate chain. But that is different than relying on a pre-existing (direct) trust relationship between the the AS and OP.

In other words, given a certificate chain X

AS --> trusts X cert chain
OP --> trusts X cert chain

But this does not imply

AS --> trusts OP

Reply:
trust is in this case transitive. It is probably best to clarify this point in the text.

Reply:
Captured in https://github.com/solid/solid-oidc/issues/66
We can discuss it separately since this PR didn't introduce it.